README.md

```diff
@@ -84,7 +84,7 @@ NEW_VERSION="1.0.1"
 git checkout master
 git checkout -b version-update-v$NEW_VERSION
 
-sed -i 's|DiathekeVersion = "[0-9.]*"|DiathekeVersion = "'$NEW_VERSION'"|g' grpc/swift-diatheke/Diatheke.swift
+sed -i 's|DiathekeSDKVersion = "[0-9.]*"|DiathekeSDKVersion = "'$NEW_VERSION'"|g' grpc/swift-diatheke/Diatheke.swift
```

Comment:
@jhollowayj this is where the `DiathekeSDKVersion` changes when we tag a new version